openslides/motions/static/js/motions/site.js

```diff
@@ -1435,10 +1435,7 @@ angular.module('OpenSlidesApp.motions.site', [
         };
         // Export dialog
         $scope.openExportDialog = function (motions) {
-            ngDialog.open(MotionExportForm.getDialog(motions));
-        };
-        $scope.pdfExport = function (motions) {
```

Comment:
Please do not delete this function. It is used for a non admins. Those users do not get the dialog but just an export with default settings.